### ISSUE TITLE
Next Button Accessibility Issue

### ISSUE THREAD

**Issue by jeannekitchens** [t3-innovation-network/desm#368]
From @siuc-nate

Joe, as a standard the site needs to meet accessibility standards.  This should be addressed. 

The "next" button's accessibility should  be improved:
- It should be the next thing in the tab order after the last item in the form (currently it is the first thing in the tab order for the form)
- The outline that shows around it is not very easy to see

![image](https://user-images.githubusercontent.com/20917206/206202242-60f6390f-1bdb-451d-bf74-a105da963c76.png)

**Comment by jbaird123** [comment]
@excelsior - Please fix the tab order and enhance the border around the button.

**Comment by jbaird123** [comment]
Resolved.  Tab order fixed and border enhanced for clarity.
